spec/index.html

```diff
@@ -1466,164 +1473,233 @@ <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
 <section id="section-classic-full-interop" class="informative">
   <h2>Interoperability between RDF [=Classic=] and RDF [=Full=]</h2>
 
-  <p>This section provides transformations between RDF [=Classic=] datasets and RDF [=Full=] datasets,
+  <p class=issue>Should we make this section normative?</p>
+
+  <p>This section provides transformations between [=Full=] [=RDF graphs=] (resp. [=RDF datasets=]) and [=Classic=] [=RDF graphs=] (resp. [=RDF datasets=]),
   to provide some level of interoperability between the different classes of <a href="#conformance">Conformance</a>.
 
   <p class=issue>Should we go even further and aim to provide interoperability between <em>RDF 1.1</em> and RDF 1.2 [=Full=]?</p>
 
-  <p>It defines the <a href="#section-unstar-algo">`unstar`</a> algorithm, which transforms an RDF [=Full=] dataset into an RDF [=Classic=] dataset by encoding all triple-terms into dedicated named graphs.
-    This algorithm is designed to be:
-  </p>
+  <p class=issue>AT RISK: the working group may decide to replace the terms `rdf:TripleTerm`, `rdf:ttSubject`, `rdf:ttPredicate` and `rdf:ttObject` used in this section by other terms, possibly in a different namespace.</p>
+
+  <p>These transformation are designed to be:</p>
 
   <dl>
     <dt>Information preserving</dt>
-    <dd>It must be possible to reconstruct the input dataset from the output dataset.
-      Note that, on the other hand, the algorithm is not designed to be semantics preserving:
-      the graphs in the produced dataset are not semantically <a>equivalent</a> to their corresponding graph in the input dataset.
+    <dd>It must be possible to reconstruct the input graph (resp. dataset) from the output graph (resp. dataset).
+      Note that, on the other hand, these transformations are not designed to be semantics preserving:
+      the output graph is not semantically [=equivalent=] to the input graph,
+      at least not in the entailment regimes defined in [[RDF12-SEMANTICS]].
     </dd>
     <dt>Idempotent</dt>
-    <dd>Transforming a dataset that is already complying with RDF [=Classic=] (i.e. containing no <a>triple term</a>) must result in the same dataset.
+    <dd>Applying a transformation several times to a graph (resp. dataset) should have the same effect as applying it once.
+      Moreover, [=classicizing=] a graph (resp. dataset) that is already complying with RDF [=Classic=] (i.e. containing no [=triple term=]) must result in the same graph (resp. dataset).
```

Comment:
I think we will be well served by using `RDF Classic` as the defined term, rather than just `Classic`. I'm not making that change here; only suggesting it to you.
```suggestion
      Moreover, [=classicizing=] a graph (resp. dataset) that is already complying with RDF [=Classic=] (i.e., containing no [=triple term=]) must result in the same graph (resp. dataset).
```